README.md

```diff
@@ -126,8 +126,8 @@ gnag {
         repoName 'btkelly/repo'
         authToken '0000000000000'
         issueNumber '1'
-        commentInline true
-        commentOnSuccess true
+        setCommentInline true
+        setCommentOnSuccess true
```

Comment:
There's no `commentInline` method as we used to do for Groovy-only buildscripts. I think standardizing on getters/setters and reducing the amount of extra we need to do to support both Groovy and Kotlin makes sense.